### DISCUSSION TITLE
Is it normal?

### DISCUSSION THREAD

**Discussion by xchl** [awcodes/filament-tiptap-editor#209] [Q&A] [answered]
https://github.com/awcodes/filament-tiptap-editor/assets/3870041/a4848362-1f01-405c-92c7-1874a2b3e0c2

It looks like clicking the button causes the focus to be gained and lost repeatedly.

**Comment by awcodes** [comment, marked as answer]
For the time being, it is normal, because clicking button removes the focus from the editor.

**Comment by awcodes** [comment]
Latest version should be better. Make sure to rebuild your custom theme adapter updating.

**Reply by xchl** [reply]
Yes, it's better. I've already tried it. Thanks!
